.github/workflows/e2e-kind.sh

```diff
@@ -16,8 +16,8 @@ readonly KONG_IMMUNITY_REGISTRY=kong-docker-kong-brain-immunity-base.bintray.io
 readonly KONG_IMMUNITY_IMAGE=kong-brain-immunity
```

Comment:
These values here need to reflect the image registry and path

Reply:
The build only takes 3 minutes to see in the lint-test job what went wrong in this case `kong-brain-immunity:4.0.0 not found`